docs/sources/dashboards/build-dashboards/view-dashboard-json-model/index.md

```diff
@@ -146,15 +159,18 @@ The grid has a negative gravity that moves panels up if there is empty space abo
 
 Usage of the fields is explained below:
 
-| Name                  | Usage                                  |
-| --------------------- | -------------------------------------- |
-| **collapse**          | whether timepicker is collapsed or not |
-| **enable**            | whether timepicker is enabled or not   |
-| **notice**            |                                        |
-| **now**               |                                        |
-| **refresh_intervals** |                                        |
-| **status**            |                                        |
-| **type**              |                                        |
+| Name                  | Usage                                                     |
+| --------------------- | --------------------------------------------------------- |
+| **collapse**          | whether timepicker is collapsed or not                    |
+| **enable**            | whether timepicker is enabled or not                      |
+| **notice**            |                                                           |
+| **now**               |                                                           |
+| **hidden**            | whether timepicker is hidden or not                       |
+| **nowDelay**          | exclude recent data that may be incomplete                |
```

Comment:
In this other [PR](https://github.com/grafana/grafana/pull/79703/files#diff-b468897532069e2eea891a159f8136398cb7a9869a17d8f8ae376fdaf2800022R203) we just added a better description of this property in our dashboard schema, this can be used here :) 


```suggestion
| **nowDelay**          | override the now time by entering a time delay. Use this option to accommodate known delays in data aggregation to avoid null values.               |
```